test/object-store/sync/flx_sync.cpp

```diff
@@ -4657,8 +4659,10 @@ TEST_CASE("flx sync: no garbage data if wait for download prior to wait subs com
     }
     SECTION("synchronous wait") {
         auto new_sub = mut_subs.commit();
```

Comment:
nit: you can commit the subscription and wait for download outside the sections since it's common to both tests.